qiskit/circuit/classicalfunction/__init__.py

```diff
@@ -25,7 +25,7 @@
 how to synthesize a simple boolean function defined using Python into a
 QuantumCircuit:
 
-   .. code-block::
+  .. code-block:: python
```

Comment:
There's a space remove here, but there's still leading indentation - is that what we expect, and is it displaying correctly?

Reply:
Docs builds before and after render identically (the conversion script currently ignores the specified language, see https://github.com/Qiskit/documentation/issues/2162). I think I removed the space to keep indentation a multiple of 2.